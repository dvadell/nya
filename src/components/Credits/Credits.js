import React  from 'react'
import translations from './Credits.translations'
import Localizer from '../../lib/Localizer'

const Localize = Localizer(translations)

const Credits = () => {
    return (<div>
                <h1><Localize>Credits</Localize></h1>
                <p>Diego Vadell - <Localize>Programmer</Localize></p>
                <p>Ver√≥nica Caligaris - Project Manager / QA</p>
                <p>Black Cat Line, <Localize>by</Localize> George Bokhua</p>
            </div>)
} 

export default Credits