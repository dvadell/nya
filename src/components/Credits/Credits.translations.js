const translation = {
    'Credits': {
        es_ar:  'Créditos',
        es_es:  'Créditos',
        en:     'Credits'
    },    
    'Programmer': {
        es_ar:  'Programador',
        es_es:  'Programador',
        en:     'Programmer'
    },    
    'by': {
        es_ar:  'de',
        es_es:  'de',
        en:     'by'
    }
}

export default translation;