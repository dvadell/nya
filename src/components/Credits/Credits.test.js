import { shallow } from 'enzyme'
import React from 'react';
import Credits from './Credits';

describe('Credits', () => {
    it('expect to render empty Credits component', () => {
    expect(shallow(<Credits />)).toMatchSnapshot()
    })
})

