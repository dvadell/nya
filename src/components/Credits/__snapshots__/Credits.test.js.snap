// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Credits expect to render empty Credits component 1`] = `ShallowWrapper {}`;
